embeds/million-thrasher/_source/main.js

```diff
@@ -0,0 +1,126 @@
+(function() {
+
+  var isCont;
+
+  var isUK;
+  var isAU;
+
+  var thanks;
+  var sell;
+  var subcribeButtonSell;
+  var subcribeButtonThanks;
+  var dots;
+  var dotsTwo;
+
+  var allSell;
+  var allThanks;
+  var ausOnlySell;
+  var ausOnlyThanks;
+
+
+  function loadThrasherElements() {
+      // Checks if a contriubtor or not
+      isCont = document.cookie.split(';').map(a => a.trim()).map(a => a.split('=', 2)).filter(a => ['gu_paying_member', 'gu_digital_subscriber', 'gu_recurring_contributor', 'gu.contributions.contrib-timestamp'].includes(a[0])).filter(a => a[1] != 'false').length != 0
+
+      // Checks if in UK
+      isUK = ["GB"].includes(JSON.parse(localStorage.getItem('gu.geolocation')).value);
+      isAU = ["AU"].includes(JSON.parse(localStorage.getItem('gu.geolocation')).value);
+
+      thanks = document.getElementById("million-wrap-thanks");
+      sell = document.getElementById("million-wrap-sell");
+      subcribeButtonSell = document.getElementById("uk-only-sell");
+      subcribeButtonThanks = document.getElementById("uk-only-thanks");
+      dots = document.getElementById("dots-one");
+      dotsTwo = document.getElementById("dots-two");
+
+      allSell = document.getElementById("all-sell");
+      allThanks = document.getElementById("all-thanks");
+      ausOnlySell = document.getElementById("aus-only-sell");
+      ausOnlyThanks = document.getElementById("aus-only-thanks");
+
+      // If a contributor, show appropriate thrahser
+      if (isCont == true) {
+        thanks.classList.add("show");
+        console.log('Is a contributor');
```

Comment:
Minor: Do we need to still console log here and below as the thrasher loads?

<img width="476" alt="screen shot 2018-11-12 at 16 50 18" src="https://user-images.githubusercontent.com/8607683/48362225-16b3c680-e69b-11e8-974a-4b7332bcd48d.png">

😨 I feel a little watched! 🔍

Reply:
Gone ✨